jax/numpy/lax_numpy.py

```diff
@@ -1681,6 +1681,8 @@ def _pad(array, pad_width, mode, constant_values):
 
 @_wraps(onp.pad)
 def pad(array, pad_width, mode='constant', constant_values=0):
+  if isinstance(pad_width, list):
```

Comment:
Could you unconditionally call `tuple(pad_width)`? Shorter and maybe even faster.

Reply:
That’s what I had before, but `pad_width` can apparently be a Python/NumPy scalar as well.

Reply:
`pad_width` should be a tuple of tuples. Here we enforce only the top-level to be a tuple, but we still allow tuple of lists.